src/app/counter-analytics/day-of-week-histogram/day-of-week-histogram.page.component.html

```diff
@@ -0,0 +1,15 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button icon="arrow-back" defaultHref="/counter-analytics/{{ counter.id }}"></ion-back-button>
+    </ion-buttons>
+    <ion-title>
+      <ion-icon name="analytics"></ion-icon>
+      {{ counter.title }}
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <div id="chart-container" #chartContainer></div>
```

Comment:
do we need both the `id="chart-container"` and the `#chartContainer` here?

Reply:
As I remember the CSS ID and the ID to access the element from the TypeScript code are not coupled. But I will check again.

Reply:
`#chartContainer` is not recognized in CSS.